### PR TITLE
Fixes #79 - Remove h3 from accordion example

### LINKED ISSUE #79: Remove h3 from accordion example
## Problem/Motivation
Collapse example on the documentation site is using `h3`s, which is causing margin issues.

### Describe the bug
Margin inside collapse headers, which doesn't need to be there.
Screen Shot 2020-05-08 at 10 23 39 AM

### To Reproduce
View the accordion example on the docs site, note the margin-top on the header due to the use of an `h3` tag.

## Proposed resolution
Remove the `h3` from the documentation example.

### Expected behavior
No margin.

### PR DESCRIPTION
**Pull request by akslay**
Fixes #79 by removing the h3 from the accordion example, which removes the excess margin that shouldn't be there.

### CODE REVIEW

**Comment by akslay** [comment]
Bug: https://review.digital.arizona.edu/arizona-bootstrap/master/docs/0.0/components/collapse/#accordion-example

Fix: https://review.digital.arizona.edu/arizona-bootstrap/80/merge/docs/0.0/components/collapse/#accordion-example

File: site/docs/0.0/components/collapse.md [hunk 1]
```diff
@@ -91,11 +89,9 @@ Using the [card]({{ site.baseurl }}/docs/{{ site.docs_version }}/components/card
   </div>
   <div class="card">
     <div class="card-header" id="headingTwo">
-      <h3 class="mb-0">
```
Comment (trackleft) [inline site/docs/0.0/components/collapse.md:94]: I think we should add a text-size-h3 class

Comment (trackleft) [reply site/docs/0.0/components/collapse.md:94]: Also, we should consider how screen readers and other semantics will be affected..

Comment (trackleft) [reply site/docs/0.0/components/collapse.md:94]: mx-0 maybe?

**Comment by joeparsons** [comment]
Some background about a TWBS change that was made that changed the headings within Collapse examples to h2 elements:
https://github.com/twbs/bootstrap/issues/27619

**Comment by akslay** [comment]
- [ ] Add a second example including the use of headings
- [ ] Include verbiage about "depending on your use case, you may or may not need to include headings in your accordion markup"

File: site/docs/0.0/components/collapse.md (hunk 1 above)
Comment (akslay) [reply site/docs/0.0/components/collapse.md:94]: Adding back in `h3`s for the second example (follow-up items)

**Comment by trackleft** [comment]
RTBC IMHO https://review.digital.arizona.edu/arizona-bootstrap/bug/79/docs/0.0/components/collapse/#accordion-example

**Pull request merged**